<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuiRepeater</class>
 <widget class="QWidget" name="GuiRepeater">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HTTP(S) Gui Client</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <layout class="QVBoxLayout" name="requestVLayout">
     <item>
      <layout class="QGridLayout" name="editRequestLayout">
       <item row="5" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <spacer name="requestButtonsVSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="addRequestHeaderButon">
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="deleteRequetHeaderButton">
             <property name="text">
              <string>Delete</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Maximum</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="deleteAllRequestHeaderButton">
             <property name="text">
              <string>Delete All</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableView" name="requestHeadersTableView">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>1</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="requestHeadersLabel">
         <property name="text">
          <string>Request Headers:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="requestBodyLabel">
         <property name="text">
          <string>Request Body:</string>
         </property>
         <property name="buddy">
          <cstring>requestBodyPlainTextEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QPlainTextEdit" name="requestBodyPlainTextEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>2</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <layout class="QGridLayout" name="startingLineGridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="requestMethodLabel">
           <property name="text">
            <string>Method</string>
           </property>
           <property name="buddy">
            <cstring>requestMethodComboBox</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="urlLineEditLabel">
           <property name="text">
            <string>URL</string>
           </property>
           <property name="buddy">
            <cstring>urlLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QComboBox" name="requestMethodComboBox">
           <property name="currentText">
            <string notr="true"/>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="urlLineEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="2">
    <layout class="QVBoxLayout" name="replyVLayout">
     <item>
      <layout class="QHBoxLayout" name="replyStatusHLayout">
       <item>
        <widget class="QLabel" name="replyStatusStatusLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Reply Status:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="replyStatusCodeDescriptionLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="replyHeadersLabel">
       <property name="text">
        <string>Reply Headers:</string>
       </property>
       <property name="buddy">
        <cstring>replyHeadersTableView</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="replyHeadersTableView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="replyBodyLabel">
       <property name="text">
        <string>Reply Body:</string>
       </property>
       <property name="buddy">
        <cstring>replyBodyPlainTextEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="replyBodyPlainTextEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>2</verstretch>
        </sizepolicy>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QPushButton" name="sendRequestButton">
     <property name="text">
      <string>Send Request</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="fuzzButton">
     <property name="text">
      <string>Fuzzing...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
